elcid/assets/js/elcid/controllers/clinical_advice_form.js

```diff
@@ -25,6 +25,7 @@ angular.module('opal.controllers').controller(
             self.save = function(){
               ngProgressLite.set(0);
               ngProgressLite.start();
+              item.episode = $scope.episode;
```

Comment:
Does this work ? :D 

There are definitely no e.g. trailing other references to episode in closures inside item?

Reply:
I updated it so there weren't any, it is something that I was worried was fragile, but the unit tests cover it so if we change the opal side they will break.